dojo/tools/checkmarx_one/parser.py

```diff
@@ -81,6 +80,8 @@ def parse_iac_vulnerabilities(
                         date = self._parse_date(instance.get("firstDetectionDate"))
                     else:
                         date = self._parse_date(instance.get("lastDetectionDate"))
+                    instance_details = self.determine_state(instance)
+                    instance_details.update(base_finding_details)
```

Comment:
Should `instance_details` be referenced on line 94 instead of `base_finding_details`, as is done on line 186? (Can't make a suggestion since that line's not in the diff)

https://github.com/DefectDojo/django-DefectDojo/blob/84b90bea7d413dc2ea3bd842f966a2d967d04f58/dojo/tools/checkmarx_one/parser.py#L86-L95

https://github.com/DefectDojo/django-DefectDojo/blob/84b90bea7d413dc2ea3bd842f966a2d967d04f58/dojo/tools/checkmarx_one/parser.py#L181-L187

Reply:
Good catch, you are right here